### PR TITLE
ENYO-977 Next cannot be called during onTransitionFinish

### PR DESCRIPTION
**Pull request by webOS101**
Enyo-DCO-1.1-Signed-off-by: Roy Sutton <roy.sutton@lge.com>

### CODE REVIEW

File: panels/source/Panels.js [hunk 1]
```diff
@@ -711,9 +711,13 @@
...
 
 			if (fire) {
-				this.fireTransitionFinish();
+				this.fireTransitionFinish(true);
+			} else {
+				this.clearTransitionData();
 			}
+		},
 
+		clearTransitionData: function() {
```
Comment (aarontam) [inline panels/source/Panels.js]: @webOS101 should we add some documentation for this method?

Comment (webOS101) [inline panels/source/Panels.js]: Hmm, yeah, we probably should at least mark it private.

Comment (webOS101) [inline panels/source/Panels.js]: Done.

Comment (aarontam) [inline panels/source/Panels.js]: :+1:

**Pull request merged**